index.html

```diff
@@ -0,0 +1,1753 @@
+              </path>
+            </g>
+          </svg>
+        </a>
+        <div class="col p-0">
+          <div class="row justify-content-end align-content-center text-uppercase font-weight-bold pr-0">
+            <a class="text-decoration-none text-white px-2 pt-2" href="#">Sign in</a>
+            <a class="text-decoration-none px-2 pt-2" style="background: white; color:#f72210; " href="#">Subscribe <i
+                class="fas fa-chevron-right pl-1"></i></a>
+            <button class="navbar-toggler p-0 ml-2" type="button" data-toggle="collapse" data-target="#collapsingNavbar">
+              <span class="navbar-toggler-icon pt-2" style="color: white;"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="collapsingNavbar">
+              <ul class="style navbar-nav float-right  mt-3" style="margin-right: -1em; background: white;">
+                <li>
+                  <form class="form-inline" action="#">
+                    <input class="form-control mr-1" type="text" placeholder="Search" />
+                  </form>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link text-dark" href="#">U.S.</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link text-dark" href="#">World</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link text-dark" href="#">Business</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link text-dark" href="#">Link</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link text-dark" href="#">Business</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link text-dark" href="#">Tech & Science</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link text-dark" href="#">Culture</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link text-dark" href="#">Newsgeek</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link text-dark" href="#">Sports</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link text-dark" href="#">Health</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link text-dark" href="#">Opinion</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link text-dark" href="#">Vintage</a>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+    </nav>
+    
+    <div class="container-fluid  d-lg-block d-md-none d-sm-none " style="background-color: rgba(244, 245, 246, 0.973);">
```

Comment:
**[REQUIRED]** Remove inline styles. Add all your styles to the external CSS file.